chapter_computer-vision/image-augmentation.md

```diff
@@ -359,12 +359,17 @@ def train_ch13(net, train_iter, test_iter, loss, trainer, num_epochs,
 #@tab pytorch
 #@save
 def train_ch13(net, train_iter, test_iter, loss, trainer, num_epochs,
-               devices=d2l.try_all_gpus()):
-    """Train a model with mutiple GPUs (defined in Chapter 13)."""
+               devices=d2l.try_all_gpus(), lazy=False):
```

Comment:
Let's try avoiding the extra `lazy` arg if possible.

Reply:
Thanks for the suggestion. I have removed this argument.